dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -246,5 +248,14 @@ private static boolean isConnected(final Connection conn) throws SQLException {
         return (conn != null && !conn.isClosed());
     }
 
+    private Map<Class<?>, TypeSpecifier> typeSpecifiers = new HashMap<Class<?>, TypeSpecifier>();
+
+    public TypeSpecifier getTypeSpecifier(Class<?> targetClass) {
+    	return typeSpecifiers.get(targetClass);
+    }
```

Comment:
Can we make it without adding new methods to `DBEnvironment`? If we pick the change which we had in the Informix PR - the one that moves statement creation in the environment, then we won't need to expose these specifier accessors as public API.